src/apps/chifra/internal/daemon/handle_rpc_integration_test.go

```diff
@@ -0,0 +1,66 @@
+package daemonPkg
+
+import (
+	"context"
+	"net"
+	"testing"
+
+	"github.com/TrueBlocks/trueblocks-core/src/apps/chifra/proto"
+	grpc "google.golang.org/grpc"
```

Comment:
Just curious. Is this actually required? It's not in the other file above.

Reply:
It wasn't, good catch